addons/mrp/models/mrp_workorder.py

```diff
@@ -83,6 +84,7 @@ class MrpWorkorder(models.Model):
     duration_percent = fields.Integer(
         'Duration Deviation (%)', compute='_compute_duration',
         group_operator="avg", readonly=True, store=True)
+    progress = fields.Float('Progress Done (%)', digits=(16, 2), compute='_compute_progress', store=True)
```

Comment:
@jat-odoo do you really need to store this field?

Reply:
Not really. I will make it non-stored.